e2e/server/docker-compose.yml

```diff
@@ -9,18 +9,13 @@ services:
             - e2e
 
     mongo:
-        image: mongo:4.4
+        image: mongo:6
         ports:
             - "27017:27017"
         networks:
             - e2e
-        environment:
-            - MONGO_INITDB_DATABASE=superdesk_e2e
         tmpfs:
             - /data/db
-        volumes:
-          - ./mongo-rs-initialize.js:/docker-entrypoint-initdb.d/mongo-rs-initialize.js:ro
-        command: --replSet rs0
```

Comment:
we need that for recording

Reply:
Why is replica set's required for that?
Anyway, it wasn't working on newer pymongo lib, that's why I removed it. I'm guessing might be differences in the connection string or something like that. Will investigate